spec/models/user_spec.rb

```diff
@@ -1,113 +1,129 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  let(:user) { FactoryBot.create(:user) }
-  let(:invalid_user) { FactoryBot.build(:invalid_user) }
-  let(:list) { FactoryBot.create(:list, owner_id: user.id) }
-  let(:list2) { FactoryBot.create(:list, owner_id: user.id) }
-  let(:movie) { FactoryBot.create(:movie) }
-  let(:movie2) { FactoryBot.create(:movie) }
-  let(:listing) { FactoryBot.create(:listing, list_id: list.id, user_id: user.id, movie_id: movie.id ) }
-  let(:listing2) { FactoryBot.create(:listing, list_id: list2.id, user_id: user.id, movie_id: movie2.id ) }
-
-  it { is_expected.to validate_uniqueness_of(:username) }
-  it { is_expected.to validate_presence_of(:username) }
-  it { is_expected.to have_many(:lists) }
-  it { is_expected.to have_many(:listings) }
-  it { is_expected.to have_many(:memberships) }
-  it { is_expected.to have_many(:member_lists) }
-  it { is_expected.to have_many(:member_listings) }
-  it { is_expected.to have_many(:movies) }
-  it { is_expected.to have_many(:member_movies) }
-  it { is_expected.to have_many(:tags) }
-  it { is_expected.to have_many(:taggings) }
-  it { is_expected.to have_many(:sent_invites) }
-  it { is_expected.to have_many(:reviews) }
-  it { is_expected.to have_many(:reviewed_movies) }
-  it { is_expected.to have_many(:ratings) }
-  it { is_expected.to have_many(:rated_movies) }
-  it { is_expected.to have_many(:screenings) }
-  it { is_expected.to have_many(:watched_movies) }
```

Comment:
I removed these relation specs because they're pretty pointless and just take up space.

Reply:
<img src="https://media3.giphy.com/media/l0MYzLLxlJDfYtzy0/giphy.gif"/>